archived-legacy-tests/CRSP-Tests.postman_collection.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Suggestion: Renaming the file name

Reply:
These Tests are archived - so leaving them as is